### DISCUSSION TITLE
Logging each HTTP request feature

### DISCUSSION THREAD

**Discussion by iomarmochtar** [prometheus/pushgateway#574] [Ideas]
Our pushgateway got intermittent restart, due the lack of information for what kind of job that push heavily at that time also with detail information how much the payload been sent, we fronted the pushgateway with nginx instance and it help us identity the problem from the produced log.

But i think it will be great if this http request logs is included in pushgateway and let say it will be enable if the log level is set to debug. 

If it's sounds good to be implemented, i will make a PR for that.

**Comment by beorn7** [comment]
I guess it cannot harm to log this at debug level. PR welcome.
